lector/resources/raw/io.github.BasioMeusPuga.Lector.metainfo.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 BasioMeusPuga <https://github.com/BasioMeusPuga> -->
+<!-- See <https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html> for an overview of the AppStream metainfo.xml spec. -->
+<component type="desktop-application">
+    <id>io.github.BasioMeusPuga.Lector</id>
+    <metadata_license>FSFAP</metadata_license>
+    <project_license>GPL-3.0+</project_license>
+    <name>Lector</name>
+    <summary>Ebook reader and collection manager</summary>
+
+    <description>
+        <p>
+            Lector is an ebook reader and collection manager. It offers a
+            full-screen distraction-free view, document highlighting and
+            annotations, a built-in dictionary, bookmarks, and multiple profiles for
+            changing the way the books are presented. Lector can also edit metadata,
+            so you can correct information about the books, and add keywords to make
+            them easier to find.
+        </p>
+        <p>
+            It supports the following file formats:
+        </p>
+        <ul>
+            <li>PDF</li>
+            <li>EPUB</li>
+            <li>DjVu</li>
+            <li>FictionBook (.fb2)</li>
+            <li>Mobipocket (.mobi)</li>
+            <li>Amazon Kindle (.azw, .azw3, .azw4)</li>
+            <li>Comic book archives (.cbr, .cbz)</li>
+            <li>Markdown</li>
+        </ul>
+    </description>
+
+    <launchable type="desktop-id">lector.desktop</launchable>
+
+    <screenshots>
+        <screenshot type="default">
+            <caption>Main window</caption>
+            <image>https://camo.githubusercontent.com/9e2f7346616dfdb67a77c2e6b5db148d9b4669ca/68747470733a2f2f692e696d6775722e636f6d2f35313668526b532e706e67</image>
```

Comment:
Notice I just use the URL of the existing screenshots in the main Lector repository... We would want to update these as the Lector GUI changes.